src/scrapers/platformen/scraper.py

```diff
@@ -43,6 +44,13 @@ def __init__(self, platform_url, list_endpoint, details_endpoint):
         # of some sort.
         self.list_endpoint = list_endpoint
         self.details_endpoint = details_endpoint
+        
+        httpHeaders = CaseInsensitiveDict()
+        httpHeaders['User-Agent'] = 'Helpradar.nl'
+        if headers is not None:
+            for key, value in headers.items():
+                httpHeaders[key] = value
+        self.headers = httpHeaders
```

Comment:
Using a Session with `self.headers.update()` is easier, see https://requests.readthedocs.io/en/master/user/advanced/#session-objects